components/ActivityIndicator/examples/ActivityIndicatorExample.swift

```diff
@@ -0,0 +1,85 @@
+/*
+ Copyright 2016-present the Material Components for iOS authors. All Rights Reserved.
```

Comment:
I am confused here as to what you would like me to change???

Reply:
Apologies, that wasn't very explicit.
This is a new example, and I expect you copy & pasted the license from another file.
That usually works, with the exception that we must set the copyright year to the year the file was created.
So in this case it should be 2017 instead of 2016.

Reply:
Yeah sorry for the confusion. If I have questions about the roadmap or design strategies where would be a good place to discuss that, discord app?

Reply:
Feel free to open an issue with any questions.  That can be useful if other people may also be wondering.
Someone is usually on Dischord as well and that may be a good spot for the easier ones.

Roadmap and design sound like bigger questions :)  But, we can always chat in Dischord and document the useful pieces.